Rooms/attic.json

```diff
@@ -20,6 +23,13 @@
             "table": "It's an engraved table with a mirror on top. The whole thing is painted white.",
             "chest": "It's a sturdy chest with a padlock on the front."
         },
+        "Open": {
+            "chest": {
+                "locked": "It's a sturdy chest with a padlock. It won't open.",
+                "unlocked": "There's nothing in here except a Polaroid.",
+                "empty": "There's nothing in here except a Polaroid."
```

Comment:
I think "empty" needs to be corrected.

Reply:
This part isn't functional at the moment, but changed it anyway.